hw03_coxj2.py

```diff
@@ -0,0 +1,89 @@
+#################################################################################
+# Author: John Cox
+# Username: coxj2
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Learn more about Git
+# Google Doc Link: https://docs.google.com/document/d/1T62Guh1IfBQMbXtI0dodMTa7tjEZ7ntFdILqF9CpXVw/edit?usp=sharing
+#
+#################################################################################
+
+import turtle
+import random
+
+def drawRectangle(t, l, w):
+    """
+    Draw a Rectangle at current position with 't' turtle
+    l = length, w = width
+    """
+    for i in range(2):
+        t.fd(l)
+        t.right(90)
+        t.fd(w)
+        t.right(90)
+def drawSquare(t, l):
```

Comment:
PEP8 wants some space before function definitions. Look for the gray squigglies under text. It helps with code readability. Please fix the function definitions.